plugin/src/main/java/com/denizenscript/denizen/tags/core/TextTagBase.java

```diff
@@ -362,7 +362,7 @@ public void run(ReplaceableTagEvent event) {
                   // @description
                   // Returns a special chat code that makes the following text display the input hover text when the mouse is left over it.
                   // This tag must be followed by an <&end_hover> tag.
-                  // Optionally specify the hover type as one of: SHOW_TEXT, SHOW_ACHIEVEMENT, SHOW_ITEM, or SHOW_ENTITY.
+                  // Available hover types: SHOW_TEXT, SHOW_ACHIEVEMENT, SHOW_ITEM, or SHOW_ENTITY.

```

Comment:
All the random phrasing changes here seem... odd. Like... why?

Reply:
Because in that particular tag, it is not OPTIONAL. we have two sets of meta there, one for it with, and one without.

Reply:
Also.. the click isn't a hover, so copy-meta